InventoryInStorePickupAdminUi/Test/Mftf/Test/AdminNotifyOrderIsReadyForPickupTwiceWithSimpleProductCustomStockTest.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminNotifyOrderIsReadyForPickupTwiceWithSimpleProductCustomStockTest">
+        <annotations>
```

Comment:
Test case link seem to be missing. Please add a test case and link before merging